plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -792,6 +791,38 @@ public void testAddColumnAndVacuum()
         }
     }
 
+    @Test
+    public void testDropNotNullConstraintWithColumnMapping()
+    {
```

Comment:
Tiny detail relevant probably for io.trino.testing.BaseConnectorTest#testDropNotNullConstraint

Pls add an extra `DROP NOT NULL` when the column is nullable to ensure that we can't drop more than once a not null constraint.

Reply:
That's already ensured by engine and covered by `TestDropNotNullConstraintConstraintTask#testDropNotNullConstraintNullableColumn`